documentation/src/main/archive/1.0.0-performance-tests/PerformanceTests.md

```diff
@@ -0,0 +1,5 @@
+# Performance Tests
```

Comment:
Should we modify the archive documentation?

Reply:
I've moved those files (1.0.0 performance results) there (to `archive/1.0.0-performance-tests`) to not loose them.